nbodykit/cosmology/power/__init__.py

```diff
@@ -1,4 +1,5 @@
+from .galaxy import GalaxyPower
```

Comment:
Let's also add a unit test or two that exercises this file,, e.g., in this file: https://github.com/bccp/nbodykit/blob/a387cf429d8cb4a07bb19e3b4325ffdf279a131e/nbodykit/cosmology/tests/test_power.py